include/multipass/cli/format_utils.h

```diff
@@ -135,8 +136,15 @@ Details multipass::format::sort_instances_and_snapshots(const Details& details)
         if (a.has_instance_info())
             return a.name() < b.name();
         else
+        {
+            if (a.name() < b.name())
+                return true;
+            else if (a.name() > b.name())
+                return false;
+
             return TimeUtil::TimestampToNanoseconds(a.snapshot_info().fundamentals().creation_timestamp()) <
                    TimeUtil::TimestampToNanoseconds(b.snapshot_info().fundamentals().creation_timestamp());
+        }
```

Comment:
You can invert the logic to simplify a little:
```suggestion
        {
            if (a.name() == b.name())
                return TimeUtil::TimestampToNanoseconds(a.snapshot_info().fundamentals().creation_timestamp()) <
                   TimeUtil::TimestampToNanoseconds(b.snapshot_info().fundamentals().creation_timestamp());
            return a.name() < b.name();
        }
```